fog-core.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/fog/fog-core"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
+  spec.files         = `git ls-files -z`.split("\x0")
```

Comment:
FYI - `$/` was part of the orignal Bundler generated template.

`$INPUT_RECORD_SEPARATOR` is defined in the standard `English` library and was replaced by Rubocop to avoid having Perlism like `$/`

Presently `bundle gem` generates this line as:

```
spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```

You can either `require "English"` for these readable version to be defined or replace it with the latest Bundler line.